babadee001/HelloBooks
# implement feedback

#### What does this PR do?
Implement feedback given from others' score cards
#### Description of Task to be completed?
- Fix wrong file naming conventions
- Update README file for license and contribution
- Include .env example file
- Setup environment variables in webpack environment plugin
#### How should this be manually tested?
- Open the repo
- Check naming conventions
- Check .env.example file
- Check webpack for the environment plugin modification
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
NA
#### Screenshots (if appropriate)
NA
#### Questions:
NA
